wagtailmarkdown/utils.py

```diff
@@ -154,6 +155,10 @@ def _get_markdown_kwargs():
 
     if hasattr(settings, "WAGTAILMARKDOWN_EXTENSIONS"):
         markdown_kwargs["extensions"] += settings.WAGTAILMARKDOWN_EXTENSIONS
+        warnings.warn(
+            "WAGTAILMARKDOWN_EXTENSIONS will be deprecated in version 7.1, use WAGTAILMARKDOWN = { extensions: {} } as dict instead",
```

Comment:
```suggestion
            "WAGTAILMARKDOWN_EXTENSIONS will be deprecated in version 0.9.0, use WAGTAILMARKDOWN = { extensions: {} } as dict instead",
```

My plan is to get this in v0.7.0, so giving it one extra release for the deprecations to be addressed

Reply:
I will do..